### PR TITLE
KLBP-282: Breadcrumbs & Workflow status update on Step 1.

### PR DESCRIPTION
**Pull request by ioan-iacob-netea**

### CODE REVIEW

**Review by andreir-suciu — APPROVED** [review]

File: src/hooks/useBreadcrumbsHook.ts [hunk 1]
```diff
@@ -1,10 +1,47 @@
...
+import { IBreadcrumb } from '../interfaces/Breadcrumb'
 
 export const useBreadcrumbsHook = () => {
   const { step } = useSelector((state: IStore) => state.breadcrumbReducer)
   const { getNextStep } = useStepHook()
+  const dispatch = useDispatch()
+  const { request } = kycCore
+
+  const getBreadcrumbs = async () => {
+    const breadcrumbData = (await kycCore.breadcrumb.getBreadCrumb()) as IBreadcrumb[]
```
Comment (mariuspop86) [inline src/hooks/useBreadcrumbsHook.ts:16]: Please dispatch a loading here

File: src/hooks/useBreadcrumbsHook.ts [hunk 2]
```diff
@@ -1,10 +1,47 @@
...
+    dispatch(breadcrumbActions.breadcrumbReceived(breadcrumbData))
+
+    return breadcrumbData
+  }
+
+  // TODO: Move patchBreadcrumbs to KYC-Core
+  const patchBreadcrumbs = async (isPrevious: boolean = false, isAfterLogin: boolean = false) => {
```
Comment (mariuspop86) [inline src/hooks/useBreadcrumbsHook.ts:23]: I think we should use constants here to improve code readability

**Review by iuliudascalu — APPROVED** [review]

File: src/hooks/useBreadcrumbsHook.ts (hunk 1 above)
Comment (ioan-iacob-netea) [reply src/hooks/useBreadcrumbsHook.ts:16]: Resolved

File: src/hooks/useBreadcrumbsHook.ts (hunk 2 above)
Comment (ioan-iacob-netea) [reply src/hooks/useBreadcrumbsHook.ts:23]: Updated

**Review by iuliudascalu — APPROVED** [review]

File: src/hooks/useBreadcrumbsHook.ts [hunk 3]
```diff
@@ -1,15 +1,64 @@
...
+      next_step: step + 1,
+      step,
+    }
+  }
 
   // Method return breadcrumb step numbers instead of their indexes in order to ensure to correct update
   const nextBreadcrumbs = () => {
     return {
-      next_step: getNextStep() + 1,
+      next_step: step + 2,
```
Comment (mariuspop86) [inline src/hooks/useBreadcrumbsHook.ts:61]: Are you sure here we should not keep this `next_step: getNextStep() + 1,`

**Review by andreir-suciu — APPROVED** [review]

**Review by mariuspop86 — APPROVED** [review]

File: src/hooks/useBreadcrumbsHook.ts (hunk 3 above)
Comment (ioan-iacob-netea) [reply src/hooks/useBreadcrumbsHook.ts]: Reverted change.

**Pull request merged**